src/Catalog/Helpers/DB2CatalogCursor.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Data.SqlTypes;
+
+namespace NuGet.Services.Metadata.Catalog.Helpers
+{
+    public sealed class DB2CatalogCursor
```

Comment:
nit: since this is used only by GalleryDatabaseQueryService it can be an internal class inside it

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/ca3107303239c439127b30fb3b1c9ba507c64843